<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/rtr.png</normaloff>:/images/rtr.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Saves the selected settings to your preferences file and closes the window.</string>
   </property>
   <property name="text">
    <string>Save settings</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>211</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Preferences settings</string>
   </property>
   <widget class="QCheckBox" name="AUTO_SAVE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Disables the automatic save function.</string>
    </property>
    <property name="text">
     <string>Disable autosave</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="SHOW_BANNERS">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Disables the giant floating banners above units for their identification.</string>
    </property>
    <property name="text">
     <string>Hide floating banners</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="DISABLE_ARROW_MARKERS">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Disables the arrow markers on the ground when units are selected (the units will only slightly glow when selected).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Disable arrow markers</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MINIMAL_UI">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The minimal UI is an alternative user inferface for battles that reduces clutter and the space used by the interface.</string>
    </property>
    <property name="text">
     <string>Minimal UI</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="UNIT_EXPERIENCE_UPGRADE_EFFECT">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Disables the bright graphics effects whenever a unit acquires a level of experience in the battlefield.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Disable veterancy graphic effects</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="UNLIMITED_MEN_ON_BATTLEFIELD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If disabled, battles will not have more than 20 units per side, and any reinforcements that make the number of units higher than 20 are delayed by the game until enough slots become available.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Unlimited troops on the battlefield</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="defaultsButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Resets all the preferences to their default settings.</string>
   </property>
   <property name="text">
    <string>Reset defaults</string>
   </property>
  </widget>
  <widget class="QPushButton" name="discardButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Saves the selected settings to your preferences file and closes the window.</string>
   </property>
   <property name="text">
    <string>Discard changes
and close</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>saveButton</zorder>
  <zorder>defaultsButton</zorder>
  <zorder>discardButton</zorder>
 </widget>
 <tabstops>
  <tabstop>saveButton</tabstop>
  <tabstop>SHOW_BANNERS</tabstop>
  <tabstop>DISABLE_ARROW_MARKERS</tabstop>
  <tabstop>UNIT_EXPERIENCE_UPGRADE_EFFECT</tabstop>
  <tabstop>MINIMAL_UI</tabstop>
  <tabstop>UNLIMITED_MEN_ON_BATTLEFIELD</tabstop>
  <tabstop>AUTO_SAVE</tabstop>
  <tabstop>discardButton</tabstop>
  <tabstop>defaultsButton</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
